### ISSUE TITLE
Ulovlige ARIA-attributter på loginn-siden

### ISSUE THREAD

**Issue by tormoseng** [AtB-AS/webshop#247]
**Beskrivelse av bug**
Kjørte a11y-plugin til Cypress på loginn-siden som rapporterte om et problem.

Merk, dette ble funnet under en PoC av e2e-tester for webshop. PoC er foreløpig ikke sjekket inn noe sted.

**Desktop (please complete the following information):**
 - OS: Mac
 - Browser Chrome
 - Version 91

**Rapport**
```
Command:     a11y error!
Id:                   aria-allowed-attr
Impact:          serious
Tags:              (3) ["cat.aria", "wcag2a", "wcag412"]
Description:   Ensures ARIA attributes are allowed for an element's role
Help:              Elements must only use allowed ARIA attributes
Helpurl:          https://dequeuniversity.com/rules/axe/4.2/aria-allowed-attr?application=axeAPI

failureSummary: "Fix all of the following:↵  ARIA attribute cannot be used, add a role attribute or use a different element: aria-label"

html: "<div aria-label="Logg inn eller opprett en ny AtB-profil med engangskode på telefonen din. Brukere av skjermleser anbefales innlogging med e-post."><p aria-hidden="true">Logg inn eller opprett en ny AtB-profil med engangskode på telefonen din.</p></div>"

impact: "serious"
```

**Comment by mikaelbr** [comment]
Dette går egentlig inn i QA på https://github.com/AtB-AS/webshop/pull/227

**Comment by mikaelbr** [comment]
@tormoseng Sjekk ny deploya ting på #277 nå. Både at det blir rett i screen readers og med validator.

**Comment by tormoseng** [comment]
Will do

**Comment by tormoseng** [comment]
Denne er fikset av #227
